pkg/services/pluginsintegration/plugins_integration_test.go

```diff
@@ -82,7 +82,7 @@ func TestIntegrationPluginManager(t *testing.T) {
 	am := azuremonitor.ProvideService(cfg, hcp, features, tracer)
 	cw := cloudwatch.ProvideService(cfg, hcp, features)
 	cm := cloudmonitoring.ProvideService(hcp, tracer)
-	es := elasticsearch.ProvideService(hcp)
+	es := elasticsearch.ProvideService(hcp, tracer)
```

Comment:
Question for @xnyo: Can core plugins use the SDK tracer? Like in [this example](https://github.com/grafana/grafana-plugin-examples/blob/main/examples/datasource-http-backend/pkg/plugin/datasource.go#L102).

Reply:
> Can core plugins use the SDK tracer?
@andresmgot No, that only works for external plugins. That's because the SDK initializes its own tracer when calling `datasource.Manage` / `app.Manage`, which doesn't happen in core plugins

Reply:
:+1: We will probably need to mock the `DefaultTracer` as we do for the [DefaultLogger](https://github.com/grafana/grafana/blob/main/pkg/plugins/backendplugin/coreplugin/registry.go#L53) so it's transparent for all plugins